docs/source/daml2js/index.rst

```diff
@@ -35,7 +35,7 @@ Here's a complete example that generates TypeScript from a project produced from
 
 - On execution of these commands:
 
-  - The directory ``my-proj/daml2js`` contains generated TypeScript and Javascript artifacts;
+  - The directory ``my-proj/daml2js`` contains generated TypeScript and JavaScript artifacts;
```

Comment:
I think this PR needs rebasing. These errors I believe aren't present on master.

Reply:
Has been rebased.